Google.Cloud.EntityFrameworkCore.Spanner/Storage/Internal/SpannerDatabaseCreator.cs

```diff
@@ -121,14 +122,11 @@ public override async Task<bool> ExistsAsync(CancellationToken cancellationToken
         {
             try
             {
-                var adminClientBuilder = new DatabaseAdminClientBuilder
-                {
-                    EmulatorDetection = EmulatorDetection.EmulatorOrProduction
-                };
-                var databaseAdminClient = await adminClientBuilder.BuildAsync(cancellationToken);
-                await databaseAdminClient.GetDatabaseAsync(_connection.DbConnection.DataSource, cancellationToken);
+                using var cmd = _connection.DbConnection.CreateCommand();
```

Comment:
Changed the `Exists` check to use a `SELECT 1` instead of the admin call `GetDatabase` to prevent unnecessary admin calls.